pettingzoo/mpe/group_spread_3p_v0.py

```diff
@@ -1,5 +1,5 @@
 from ._mpe_utils.simple_env import SimpleEnv, make_env
-from .scenarios.group_spread_one_hot import Scenario
+from .scenarios.group_spread import Scenario
```

Comment:
Seemed to be outdated code since `group_spread_one_hot.py` does not exist for me?

Reply:
I removed that file but as it was my old implementation (before we merged everything in group_spread) and missed this config. Good find